platform/mbed_retarget.h

```diff
@@ -120,14 +120,38 @@ FileHandle *mbed_target_override_console(int fd);
  * by mbed_target_override_console, else will default to serial - see
  * mbed_target_override_console for more details.
  *
- * Example:
+ * Example using UARTSerial:
  * @code
  * FileHandle* mbed::mbed_override_console(int) {
  *     static UARTSerial my_serial(D0, D1);
  *     return &my_serial;
  * }
  * @endcode
-
+ *
+ * Example using SingleWireOutput:
+ * @code
+ * FileHandle* mbed::mbed_override_console(int) {
```

Comment:
Our coding standard changed while I was writing this in the first place, and the code got updated, but not the examples. `*` binds to the right. Fix the existing one too.

Reply:
Stars are binding to the right now.